src/components/SidebarMain/SidebarMain.js

```diff
@@ -51,11 +62,11 @@ function ReturnNavLink({ value, index, url }) {
 
 // TODO: Refactor finished props... And put in own component
 /**
- * Function to render the returnMenuItems component using the ReturnNavLink component.
+ * Component to render the returnMenuItems component, which displays a list of menu items.
```

Comment:
`Component to render the returnMenuItems component, which` This part is unnecessary.
This should be enough:
`Returns a list of menu items`